surveillance-card.js

```diff
@@ -137,8 +141,8 @@ class SurveillanceCard extends LitElement {
 
     for (const camera of this.cameras) {
       const hadMotion = camera.has_motion === true;
-      const { motion_entity } = camera;
-      camera.has_motion = motion_entity in states && states[motion_entity].state === "on";
+      const { motion_entities } = camera;
+      camera.has_motion = motion_entities.some(e => this.hass.states[e].state === "on" );
```

Comment:
We need to keep the states check like it was but modify it for multiple using some.